docs/dev/adding_instruments/instrument.rst

```diff
@@ -78,7 +78,7 @@ The most basic instrument, for our "Extreme 5000" example starts like this:
     # THE SOFTWARE.
     #
 
-    # from pymeasure.instruments import Instrument
+    from pymeasure.instruments import Instrument
```

Comment:
Please remove the `testsetup` section above (lines 48-53) as this `Instrument` import overrides above definition.

Reply:
Agree. That removes a potential source of confusion. PTAL.